great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py

```diff
@@ -773,7 +773,9 @@ def _get_kl_divergence_chart(cls, partition_object, header=None):
 
     @classmethod
     def _atomic_kl_divergence_chart_template(cls, partition_object: dict) -> tuple:
-        weights = partition_object["weights"]
+        weights = (
+            partition_object.get("weights") if partition_object.get("weights") else []
+        )
```

Comment:
Could this be `weight = partition_object.get("weights", [])`?

Reply:
I don't think so. Have ternary to handle cases where the key is present but the value is None. .get will only return default value if key is missing altogether.

Reply:
hmm that's a very fair point thanks for clarifying. What are your thoughts on the following pattern:

```
weights = partition_object.get("weights", [])
if weights is None:
    weights = []
```

It is definitely more verbose but I think an argument can be made that it is a bit more explicit? I'll ultimately defer to you here!

Reply:
Ooo. I like it. More verbose but def clearer what's going on.